tests/Serializer/BaseSerializationTest.php

```diff
@@ -259,9 +259,9 @@ public function testExpressionExclusionConfiguredWithDisjunctStrategy()
         $person->name = 'mike';
 
         $language = new ExpressionLanguage();
-        $language->addFunction(new ExpressionFunction('show_data', function () {
+        $language->addFunction(new ExpressionFunction('show_data', static function () {
```

Comment:
`static` ? any benefit out of it?

Reply:
Mostly performance. See https://github.com/doctrine/coding-standard/pull/76 + https://github.com/slevomat/coding-standard/issues/400.